### PR TITLE
Build cgnstools (no GUI apps)

### LINKED ISSUE #25: Packaging `CGNSTools` ?
### Comment:

I'm interested in building and packaging `CGNSTools` (see https://github.com/CGNS/CGNS/blob/4b21e4671273f661a3ac33993d7a8a5a9d12cdef/src/CMakeLists.txt#L765).

Options:
1. Just build it and ship it with the primary `cgns` package
2. Convert this to a a multi-output recipe which ships `cgns` and `cgns-tools`

Thoughts? @MicK7 @mdpiper

### PR DESCRIPTION
**Pull request by whophil**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

--- 
Attempt to build and package the optional cgnstools.
Closes #25

### CODE REVIEW

**Comment by whophil** [comment]
@conda-forge-admin please rerender

**Comment by whophil** [comment]
At first I tried to build the GUI apps, but ran into issues on non-Linux platforms. So now this only builds the non-GUI parts of CGNSTools. The CMakeLists.txt file is patched to make this work.

This is ready for review now @MicK7 @mdpiper !

**Review by mdpiper — APPROVED** [review]

**Comment by whophil** [comment]
Thanks for the review @mdpiper - any chance we can get this merged, or would you like to wait for review from @MicK7 ?

**Pull request merged**
